### DISCUSSION TITLE
Override min-width q-btn-dropdown

### DISCUSSION THREAD

**Discussion by roylans** [quasarframework/quasar#9012] [General - Components / Directives / etc]
Hi friends.

I have the following q-btn-dropdown

```
<q-btn-dropdown
  :menu-offset="styleContentMenu.menuOffset"
  :menu-anchor="styleContentMenu.menuAnchor"
  :menu-self="styleContentMenu.menuSelf"
  :dropdown-icon="styleContentMenu.icon"
  :no-icon-animation="!styleContentMenu.iconAnimation"
  :content-style="{
     'min-width':  styleContentMenu.width +'px',
     'width': styleContentMenu.width +'px',
     'height': styleContentMenu.height +'px',
     'display': 'flex',
     'justify-content': 'center'
  }"
  class="full-width full-height no-border-radius"
  no-caps flat
>
</q-btn-dropdown>

```
By default, the minimum width of the menu content is the width of the q-btn dropdown, but I need to override this value with a smaller value.

is posible to do this?

I need to set the min-width red box to 200px
![image](https://user-images.githubusercontent.com/16232825/115944841-3d8a9f80-a486-11eb-88c2-2fc138d1a72d.png)

Thanks in advance...

**Comment by metalsadman** [comment]
@roylans use `content-class` with min-width set to !important.

**Reply by roylans** [reply]
Hi @metalsadman , thanks by your reply.

```
:content-style="{
        'min-width': `${styleContentMenu.width}px !important`,
```
not working. Quasar override this value.

**Reply by metalsadman** [reply]
```css
.some-class {
  min-width: 100px !important;
}
```
```html
...
  content-class="some-class"
```

**Reply by roylans** [reply]
Hi, not working correctly.

This is the component:
![image](https://user-images.githubusercontent.com/16232825/115958008-84eb4d00-a4d3-11eb-9a18-f9cb9422ba1a.png)

This is my class
![image](https://user-images.githubusercontent.com/16232825/115958030-a1878500-a4d3-11eb-8d99-86b4c7667b28.png)

Here can you see the class is applied, but not working

![image](https://user-images.githubusercontent.com/16232825/115958099-f7f4c380-a4d3-11eb-9fed-d549350828c0.png)

Can you think of anything else
Thanks

**Reply by metalsadman** [reply]
Make a codesandbox reproduction of your code, as I've tried this in codepen and it works.
